lib/data-types.js

```diff
@@ -352,6 +352,41 @@ BLOB.prototype.toSql = function() {
 };
 
 /**
+ * Range types are data types representing a range of values of some element type (called the range's subtype). Only available in postgres.
+ * See {@link http://www.postgresql.org/docs/9.4/static/rangetypes.html|Postgres documentation} for more details
+ * @property RANGE
+ */
+
+var RANGE = function (Subtype) {
+  var options = {};
```

Comment:
Could perhaps be express more cleanly as:

```
options = _.isPlayObject(Subtype) ? Subtype : {subtype: Subtype}
```

And then generally doing a functions check on the options.subtype later, we'll want to support a non-inited `{subtype: INTEGER}` too.